paths.go

```diff
@@ -48,8 +48,7 @@ func newPaths(os, arch string, o Options) (p *Paths, err error) {
 	p.astilectronUnzipSrc = filepath.Join(p.astilectronDownloadDst, fmt.Sprintf("astilectron-%s", VersionAstilectron))
 	p.electronDirectory = filepath.Join(p.vendorDirectory, "electron")
```

Comment:
An important part of this change is to update line 49 as well I think since you also want the place where Electron is extracted to be os/arch dependent.

Something like this may do just fine:
```go
p.electronDirectory = filepath.Join(p.vendorDirectory, "electron", os, arch)
```

Also, tests would need to be updated